aicsimageio/tests/test_aics_image.py

```diff
@@ -197,3 +199,18 @@ def test_imread(resources_dir, filename, expected_shape):
 def test_channel_names(resources_dir, filename, expected_channel_names):
     img = AICSImage(resources_dir / filename)
     assert img.get_channel_names() == expected_channel_names
+
+
+def test_aicsimage_close(resources_dir):
+    filename = resources_dir / PNG_FILE
+    tempfilename = resources_dir / "temp.png"
+    shutil.copy(filename, tempfilename)
+    with pytest.raises(IOError):
+        img = AICSImage(tempfilename)
+        img.metadata
+        os.remove(tempfilename)
```

Comment:
I looked this up. It won't ever raise on Unix. 
If a Filepointer is open on *nix it has translated the name to a pointer. 
if you remove the file after this step it will nuke the file name but it the on-disk space pointed to by the pointer isn't closed until there are no pointer references into the file. 
Windows is fundamentally different.